wasm-calc11/test/Test/Typecheck/TypecheckSpec.hs

```diff
@@ -9,14 +10,22 @@ import Calc.Parser
 import Calc.Typecheck
 import Calc.Types
 import Control.Monad
-import Data.Either (isLeft)
+import Data.Bifunctor (second)
+import Data.Either (isLeft, isRight)
+import Data.FileEmbed
 import Data.Foldable (traverse_)
 import qualified Data.List as List
 import qualified Data.List.NonEmpty as NE
 import Data.Text (Text)
+import qualified Data.Text.Encoding as T
 import Test.Helpers
 import Test.Hspec
 
+-- these are saved in a file that is included in compilation
```

Comment:
Now we parse all the static files and check they typecheck, so we are able to test bigger things.